### DISCUSSION TITLE
only send iPhone 32Bit key to MQTT

### DISCUSSION THREAD

**Discussion by valbuz** [ESPresense/ESPresense#938] [Q&A] [answered]
hi
i try to find a solution to only send my 2 iPhone's (32Bit Key) to the MQTT Server.
when i fill in this keys to the filter, nothing is send to MQTT.
is there a workaround for that?

thanks
pat

**Comment by Terastar-Paperclip** [comment, marked as answer]
The filter is space-delimited, prefix-only — no wildcards or commas. See DT's explanation in [#312](https://github.com/ESPresense/ESPresense/discussions/312). Cross-linking for visibility.
